update frontend

smart-hede opened the pull request:
https://bunzz-test.atlassian.net/browse/BUNZZ1-333

smart-hede committed:
update frontend

NailKhalimov commented on README.md:
```diff
@@ -1 +1,13 @@
-# nft-marketplace-boilerplate
\ No newline at end of file
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app)
```
Please write a readme as in staking boilerplate

NailKhalimov commented on src/components/marketplace/change-price.tsx:
```diff
@@ -15,8 +17,13 @@ interface ChangePriceViewProps {
+  const [price, setPrice] = useState<number>(
+    data?.generalPrice ? data.generalPrice : 0
+  );
   console.log(tokenId, price);
```
Please delete this log

NailKhalimov commented on src/components/modal-views/context.tsx:
```diff
@@ -27,7 +27,10 @@ export function useModal() {
-  const closeModal = () => setState({ ...state, isOpen: false });
+  const closeModal = () => {
+    setState({ ...state, isOpen: false });
+    console.log(state);
```
here too

NailKhalimov commented on src/layouts/maketplace/layout.tsx:
```diff
@@ -31,21 +29,21 @@ const MarketPlaceLayout: FC<Props> = ({ children }) => {
-            />
-            <span>{Number(demoData.totalNFTs).toLocaleString()} items</span>
-          </div>
+          {/*<div className="absolute right-10 top-0 flex items-center gap-x-5 text-sm">*/}
```
Hm, why commented this part?

NailKhalimov reviewed (commented):
@smart-ticky 
Thank you!
Please check my comments and also please record a screencast of how it looks like🙏

smart-hede commented:
okay

smart-hede committed:
update readme.md

smart-hede committed:
add files didn't add

smart-hede committed:
add files didn't add

smart-hede committed:
remove console.log

smart-hede commented:
https://user-images.githubusercontent.com/68897116/222341753-64616c15-f711-4c60-ab2d-0242edebf1ed.mp4

NailKhalimov commented:
@smart-ticky 
Thank you! Could you add align on tabs as in the design?
Also, please check the font-weight of the fonts, seems like they are different.
<img width="63" alt="Снимок экрана 2023-03-02 в 13 36 47" src="https://user-images.githubusercontent.com/25770232/222404832-97917b7c-1117-4bad-baec-63b7c1a855fb.png">

smart-hede commented:
> Could you add align on tabs as in the design?

To confirm, you mean the sort feature?
@NailKhalimov

smart-hede committed:
fix Tab font weight and update README.md

NailKhalimov commented:
@smart-ticky 
I mean check the margin from the left side of the tabs in the design. 
Also, I've tried to connect my wallet, but it crashed, because of too many errors. Try to do the same, just use an unused wallet.
<img width="478" alt="Снимок экрана 2023-03-06 в 10 38 39" src="https://user-images.githubusercontent.com/25770232/223047634-e3c221a4-4fb7-474b-9a54-897d068fa588.png">

smart-hede committed:
fix other wallet address issue

smart-hede committed:
fix other wallet address issue

smart-hede commented:
@NailKhalimov 
fixed the error. In my side, I tried it with a new wallet and that worked very well.
and to provide sort feature, we have to build a backend source.
because we get the listing data from onchain. this is why I added the "road more" button and locked the sort feature.

smart-hede committed:
add margin for tabs

smart-hede committed:
change logo to svg and add demo sort feature

smart-hede committed:
add logo svg file

smart-hede committed:
change font color for menu items and tabs

NailKhalimov reviewed (approved):
LGTM. Thank you!
